.buildkite/pipeline.yml

```
@@ -0,0 +1,17 @@
+# SPDX-FileCopyrightText: 2021 Arthur Breitman
+# SPDX-License-Identifier: LicenseRef-MIT-Arthur-Breitman
+
+steps:
+  - label: reuse lint
+    commands:
+    - nix run -f ci.nix pkgs.reuse -c
+        reuse lint
+
+  - label: check trailing whitespace
+    commands:
+    - .buildkite/check-trailing-whitespace.sh
+
+  - label: build-ligo
+    key: build-ligo
+    commands:
+    - nix-build ci.nix -A build-ligo -o ./ligo-out/
```

I think we should already retain the produced contract as an artifact:
```suggestion
      - nix-build ci.nix -A build-ligo -o ./ligo-out/
    artifact_paths:
      - ligo-out/segmented_cfmm.tz
```

---

Added the artifact.